src/linux/py_proc.h

```diff
@@ -122,7 +130,7 @@ _py_proc__analyze_elf64(py_proc_t * self) {
   }
 
   register int symbols = 0;
-  if (p_dynsym != NULL) {
+  if (base != (Elf64_Addr) 0 - 1 && p_dynsym != NULL) {
```

Comment:
If somehow we failed to compute the ELF base address we should fail quickly, so I would put this check after `_get_base_64` has returned.

Reply:
This condition was broken as well anyway :)